app/main/views/users.py

```diff
@@ -260,13 +260,13 @@ def export_users_for_framework(framework_slug):
                 application_result = 'no result'
             else:
                 application_result = 'pass' if sf.on_framework else 'fail'
-            framework_agreement = bool(sf.serialize()['agreementReturnedAt'])
+            framework_agreement = bool(getattr(sf.current_framework_agreement, 'signed_agreement_returned_at', None))
```

Comment:
I'd have to check for both 'signed', 'countersigned' and 'on_hold' though. It seems that checking the attribute directly makes more sense in this case right now. Maybe if we change the column to show the status at some point the status will be helpful though.